src/validation.h

```diff
@@ -50,6 +51,7 @@ struct ChainTxData;
 struct DisconnectedBlockTransactions;
 struct PrecomputedTransactionData;
 struct LockPoints;
+struct CCheckpointData;
```

Comment:
No need for this forward declaration. You're including chainparams.h, which is where this is included.

Reply:
Good catch, fixed!